test-cases/artifacts/debian9.yaml

```diff
@@ -0,0 +1,18 @@
+cluster_backend: 'gce'
+gce_image: 'https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/debian-9'
+gce_instance_type_db: 'n1-standard-2'
+gce_root_disk_type_db: 'pd-ssd'
+gce_root_disk_size_db: 50
+gce_n_local_ssd_disk_db: 1
+instance_provision: "spot"
+ip_ssh_connections: 'public'
+logs_transport: 'ssh'
```

Comment:
why ssh transport is used here ?

Reply:
It was pretty tricky to run SCT with only one node(db:1, looaders:0, monitors: 0). And it failed to run with rsyslog.

Reply:
Left 'ssh' transport for internal SCT testing purpose because these tests probably the best candidates to use this log transport.